docs/configuredgraphfactory.txt

```diff
@@ -0,0 +1,442 @@
+[[configuredgraphfactory]]
+== ConfiguredGraphFactory
+
+The `ConfiguredGraphFactory` is an access point to your graphs, similar
+to the `JanusGraphFactory`. These graph factories provide static methods
+to open and close your graphs.
+
+[[overview]]
+=== Overview
+
+`JanusGraphFactory` is a class that provides an access point to your
+graphs by providing an entire Configuration each opening.
+
+`ConfiguredGraphFactory` provides an access to your graphs for which
+you have previously created configurations using the
+`ConfigurationManagementGraph`.
+
+`ConfigurationManagementGraph` allows you to manage graph configurations.
+
+`JanusGraphManager` is an internal server component that tracks
+graph references, provided your graphs are configured to use it.
+
+[[configuredgraphfactory-versus-janusgraphfactory]]
+=== ConfiguredGraphFactory versus JanusGraphFactory
+
+However, there is an important distinction between these two graph
+factories:
+
+1)  The `ConfiguredGraphFactory` can only be used if you have configured
```

Comment:
Please consistently use either "1." and "2." or "1)" and "2)" — are both of them rendered as ordered lists? "1." is probably more commonly used, and that's what it renders as, so probably the better choice to standardize on.

Reply:
In `code` sections, the ASCII doesn't render 1. and 2. as ordered lists, so Jason suggested I use 1) and 2); I think it looks fine as such. Let me have another look with fresh eyes.

Reply:
OK I updated all sections to use list ordering `1. 2. etc` except the ones with code blocks in between which will use `1) 2) etc` since the list ordering is interrupted by the code blocks.